### PR TITLE
feat: filter tokens with skip signing when validating big package

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

File: packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts [hunk 1]
```diff
@@ -1,17 +1,40 @@
...
+  dataPoints: DataPoint[],
+  allTokens: TokensConfig
+) =>
+  dataPoints.filter(
+    (dataPoint) => !allTokens[dataPoint.dataFeedId].skipSigning
+  );
+
+const countTokensWithoutSkipSigning = (allTokens: TokensConfig) =>
+  Object.values(allTokens).reduce(
+    (count, config) => (config.skipSigning ? count : ++count),
```
Comment (hatskier) [inline packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts:38]: config -> tokenConfig?

Comment (hatskier) [reply packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts:38]: I would also change `++count` to count + 1, as we don't need to increase the value of the variable here, we only need to return the new value

Comment (cehali) [reply packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts]: Done

File: packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts [hunk 2]
```diff
@@ -1,17 +1,40 @@
...
   dataPoints: DataPoint[],
-  allTokenCount?: number
+  allTokensConfig?: TokensConfig
 ) => {
-  if (!allTokenCount) {
-    throw new Error(
-      `Cannot get token count from manifest.`
-    );
+  if (!allTokensConfig) {
+    throw new Error(`Cannot get token config from manifest.`);
```
Comment (koonopek) [inline packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts:10]: Could we use `terminateWithManifestConfigError` here?

File: packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts [hunk 3]
```diff
@@ -1,17 +1,40 @@
...
   }
-  const validDataPointsPercentage = (dataPoints.length / allTokenCount) * 100;
+
+  const dataPointsWithoutSkipSigning = filterDataPointsWithoutSkipSigning(
+    dataPoints,
+    allTokensConfig
+  );
+  const tokensWithoutSkipSigningCount =
```
Comment (koonopek) [inline packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts:17]: What should happen if `tokensWithoutSkipSigningCount === 0`, should we throw or return `false`?

**Review by koonopek — CHANGES_REQUESTED** [review]

File: packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts (hunk 3 above)
Comment (cehali) [reply packages/oracle-node/src/validators/validate-data-feed-for-big-package.ts:17]: I think we should return `false` and not create big data package

**Review by koonopek — CHANGES_REQUESTED** [review]

**Review by koonopek — APPROVED** [review]

**Pull request merged**